### ISSUE TITLE
Remove CSS content from callout blocks

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#233]
From the accessibility audit, issue #232

**SC 1.1.1 Non-text content**  The callout blocks have labels ("Info", "Alert", "Warning") that are added via CSS pseudo elements. Some screenreaders do not read text contained in pseudo elements. To fix this add the text as regular text. Examples of the callout boxes can be seen on https://wpaccessibility.org/docs/topics/core/ .

**Solution**: The CSS content will be replaced by text in the content. To prevent the text looks like a heading and should be coded as a heading, the text will be added inline and bold to the callout text.

For example:

> **Alert**: This content will be reviewed and restructured. Related issue on [GitHub #122 Topic WordPress Core Accessibility Standards](https://github.com/wpaccessibility/wp-a11y-docs/issues/122).

The work:
- remove the `&::before` content from the callout CSS in _sass/components/_callouts.scss.
- find a way to add color to the bold prefix text that will replace the before content.
- chang all the callout blocks and add the prefix text.
